dashboard-all-the-things/src/app/dashboard-carousel/dynamic.component.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  AfterViewInit,
+  ChangeDetectorRef,
+  Compiler,
+  Component,
+  ComponentFactory,
+  ComponentFactoryResolver,
+  ComponentRef,
+  Input,
+  OnChanges,
+  ViewChild,
+  ViewContainerRef
+} from '@angular/core';
+import { DashboardComponent } from './../dashboards/DashboardComponent';
+import { WidgetEvent } from './../dashboards/WidgetEvent';
+import { OnDestroy } from '@angular/core';
+
+@Component({
+  selector: 'app-dynamic',
+  template: `<div #target></div>`
+})
+export class DynamicComponent implements OnChanges, AfterViewInit, OnDestroy {
```

Comment:
Yeah there was some started work on this, I'm not sure who started it but we removed it and went with the client being as dumb as possible 👍